presto-product-tests/README.md

```diff
@@ -89,16 +89,29 @@ following command:
     presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query,mysql_connector,postgresql_connector
     ```
 
-where ``<profile>`` is one of either:
+where [profile](#profile) is one of either:
 - **multinode** - pseudo-distributed Hadoop installation running on a
  single Docker container and a distributed Presto installation running on
  multiple Docker containers. For multinode the default configuration is 1 coordinator and 1 worker.
-- **singlenode** - pseudo-distributed kerberized Hadoop installation running on a
- single Docker container and a single node installation of kerberized Presto also running
- on a single Docker container.
-- **singlenode-kerberized** - pseudo-distributed Hadoop installation running on a
+- **singlenode-simple-impersonation** - pseudo-distributed Hadoop installation running on a
```

Comment:
That's a good point. I will keep the name to single node.  Anyway we are defining it